test/integration/default/controls/xcode_test.rb

```diff
@@ -10,18 +10,3 @@
     it { should_not be_symlink }
   end
 end
-
-control 'command-line-tool-sentinel' do
```

Comment:
I would leave this to test against any regressions - as we probably won't use the gem workaround forever.

Reply:
This workaround was documented [3 years ago](https://github.com/xcpretty/xcode-install/commit/a00f22ea68ad924eaa667b3236c0f75c9054bfd7), and the timeline for a permanent fix seems indefinite.

We do have a `command_line_tools` resource in the cookbook, and perhaps this should be a separate unit test for that resource specifically?

Reply:
At some point we will resume installing command line tools via some method yet to be determined, but all of those methods rely on touching this file. This test ensures that the method of installation properly removes the file - if there are tests in that specific tooling for this file then I'd say it's fine to remove.

Reply:
At this point, we can add separate tests for the `command_line_tools` resource.

Going forward, we need to upstream the tests to `build_essential`, and remove the duplication from our resource cookbook. Related to #206